### PR TITLE
User story 2v2

### LINKED ISSUE #2: User Story 2
Image

As a merchant,
When I visit my merchant dashboard (/merchants/:merchant_id/dashboard)
Then I see link to my merchant items index (/merchants/:merchant_id/items)
And I see a link to my merchant invoices index (/merchants/:merchant_id/invoices)

### PR DESCRIPTION
**Pull request by dani-wilson**
# Describe the changes below:
Add links to merchant items and merchant invoices and testing for those features

# Relevant user story:
User story 2

# Specific feedback requests? Include file name and line:
N/A

# Before submitting, check the following:
- [X] Entire test suite is passing
- [100%] SimpleCov test percentage

# Note any co-authors / anything else you'd like to add:
Close #2

### CODE REVIEW

File: app/views/dashboard/index.html.erb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <h2><div class="page-title">Merchant Dashboard</div></h2>
 
 
-<h3><%= @merchant.name %></h3>
\ No newline at end of file
+<h3><%= @merchant.name %></h3>
+<p><%= link_to "My Items", "/merchants/:merchant_id/items" %><p>
+<p><%= link_to "My Invoices", "/merchants/:merchant_id/invoices" %></p>
```
Comment (westonio) [inline app/views/dashboard/index.html.erb:6]: Just looked over US 2 and I think you might have missed the link to the dashboard: 
`When I visit my merchant dashboard (/merchants/:merchant_id/dashboard)`

Also, don't forget that you can now leverage URL pathways instead and do something like:
`merchant_dashboard_index_path`
`merchant_items_path`
`merchant_invoices_path`

Do you mind adding that last link?

File: spec/features/dashboard/index_spec.rb [hunk 2]
```diff
@@ -8,10 +8,19 @@
...
     expect(page).to have_content(@m1.name)
     expect(page).to_not have_content(@m2.name)
   end
+  #user story 2
+  it "displays a link to merchant items index and merchant invoices index pages" do
+    visit "/merchants/#{@m1.id}/dashboard"
+
+    expect(page).to have_link("My Items")
+    expect(page).to have_link("My Invoices")
+    expect(page).to_not have_link("Admin")
```
Comment (westonio) [inline spec/features/dashboard/index_spec.rb:24]: Love the extra check to ensure no links are for admin

**Review by westonio — CHANGES_REQUESTED** [review]

File: app/views/dashboard/index.html.erb (hunk 1 above)
Comment (dani-wilson) [reply app/views/dashboard/index.html.erb:6]: @westonio 
I'm looking over the US again
```
As a merchant,
When I visit my merchant dashboard (/merchants/:merchant_id/dashboard)
Then I see link to my merchant items index (/merchants/:merchant_id/items)
And I see a link to my merchant invoices index (/merchants/:merchant_id/invoices)
```
and I don't see anything about a link to the dashboard, only that I see two links FROM the dashboard.

Comment (westonio) [reply app/views/dashboard/index.html.erb:6]: Omg, @dani-wilson  I'm so sorry. My eyes played tricks on me - I think I haven't been in the best headspace today. 🤯 

This is perfect! Approving now

**Review by westonio — APPROVED** [review]

File: app/views/dashboard/index.html.erb (hunk 1 above)
Comment (dani-wilson) [reply app/views/dashboard/index.html.erb:6]: @westonio no problem! Thanks for looking over that for me, I appreciate you wanting to be thorough! :))

File: app/views/dashboard/index.html.erb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 <h2><div class="page-title">Merchant Dashboard</div></h2>
 
 
-<h3><%= @merchant.name %></h3>
\ No newline at end of file
+<h3><%= @merchant.name %></h3>
+<p><%= link_to "My Items", "/merchants/:merchant_id/items" %><p>
```
Comment (awiley33) [inline app/views/dashboard/index.html.erb:5]: nice html additions!

File: spec/features/dashboard/index_spec.rb [hunk 4]
```diff
@@ -8,10 +8,19 @@
...
+  #user story 1
   it "displays the name of each merchant" do
     visit "/merchants/#{@m1.id}/dashboard"
 
     expect(page).to have_content(@m1.name)
     expect(page).to_not have_content(@m2.name)
   end
+  #user story 2
+  it "displays a link to merchant items index and merchant invoices index pages" do
+    visit "/merchants/#{@m1.id}/dashboard"
```
Comment (awiley33) [inline spec/features/dashboard/index_spec.rb:20]: testing looks good!

**Review by awiley33 — APPROVED** [review]
Good work Dani!

**Pull request merged**